deployment/cdk-solution-helper/README.md

```diff
@@ -0,0 +1,15 @@
+## CDK Solution Helper
```

Comment:
Do we really have to package this in the repo? Can we not publish it to npm? I don't look forward to maintaining this.

Reply:
until we maintain it as a central shared artifact, this will live here